bin/publish

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+source "$(dirname "${BASH_SOURCE[0]}")/_utils.sh"
+
+
+function usage() {
+    declare script_name="$0"
+
+    echo "$script_name [-h|--help] SERVICE"
+    echo
+    echo "Options:"
+    echo "  -h|--help      : Display this help message"
+    echo
+    echo "Arguments:"
+    echo "  SERVICE: the name of the service to publish (e.g. nginx)"
+}
```

Comment:
it's the same usage of the build script no ?
it can be _usage function in _utils.sh ?

Reply:
Yup. I prefer having self sufficient scripts: `usage` and `main` functions should explicitly be part of it. It's less DRY, but it's way more understandable/explicit. Deal?